sdk/src/store.rs

```diff
@@ -1981,6 +1984,14 @@ impl Store {
         }
     }
 
+    /// check the input url to see if it is a supported remotes URI
+    pub fn is_valid_remote_url(url: &str) -> bool {
+        match url::Url::parse(url) {
+            Ok(u) => !(u.scheme() != "http" && u.scheme() != "https"),
```

Comment:
Couldn't this be simplified to `Ok(u) => u.scheme() == "http" || u.scheme() == "https"),` ?

Reply:
or u.scheme().starts_with("http"),

Reply:
The latter has a risk of accepting bogus schemes such as "httpx".